doc/devel/tag_glossary.rst

```diff
@@ -0,0 +1,181 @@
+|                            | - dense collection of short-format information                    |
+|                            | - well-defined scope, accessible information                      |
++----------------------------+-------------------------------------------------------------------+
+|``purpose: fun``            |just for fun!                                                      |
++----------------------------+-------------------------------------------------------------------+
+|``purpose: showcase``       |good showcase example                                              |
++----------------------------+-------------------------------------------------------------------+
+
+Domain tags: what discipline(s) might seek this example consistently?
+---------------------------------------------------------------------
+
+It's futile to draw fences around "who owns what", and that's not the point of domain tags. Domain tags help groups of people to privately organize relevant information, and so are not displayed publicly. See below for a list of existing domain tags. If you don't see the one you're looking for and you think it should exist, consider proposing it.
+
++-------------------------------+----------------------------------------+
+|``tag``                        | use case                               |
++===============================+========================================+
+|``domain``                     |for whom is the example relevant?       |
++-------------------------------+----------------------------------------+
+|``domain: cartography``        |                                        |
++-------------------------------+----------------------------------------+
+|``domain: geometry``           |                                        |
++-------------------------------+----------------------------------------+
+|``domain: statistics``         |                                        |
++-------------------------------+----------------------------------------+
+|``domain: oceanography``       |                                        |
++-------------------------------+----------------------------------------+
+|``domain: signal-processing``  |                                        |
++-------------------------------+----------------------------------------+
+
+Internal tags: what information is helpful for maintainers or contributors?
+---------------------------------------------------------------------------
+
++-------------------------------+-----------------------------------------------------------------------+
+|``tag``                        | use case                                                              |
++===============================+=======================================================================+
+|``internal: low bandwidth``    |allows users to filter out bandwidth-intensive examples like animations|
```

Comment:
If that is the intent then should we tag the minority (high bandwith) examples instead?